### PR TITLE
convert: remove most of the n_mult usage in convert.py

### PR DESCRIPTION
**Pull request by Green-Sky**
Little bit of clean up of `n_mult`. it is only used to calculate `n_ff` and the formula only works for llama and close derivatives. (eg not for falcon)

### CODE REVIEW

**Comment by KerfuffleV2** [comment]
Any reason not to remove the `find_n_mult` function? Nothing uses it after your changes.

**Comment by Green-Sky** [comment]
> Any reason not to remove the `find_n_mult` function? Nothing uses it after your changes.

hm true. ... time to remove my hack then :)

File: convert.py [hunk 1]
```diff
@@ -258,7 +244,7 @@ def loadOriginalParamsJson(model: LazyModel, config_path: Path) -> Params:
         n_vocab          = config["vocab_size"] if "vocab_size" in config else -1
         n_embd           = config["dim"]
         n_layer          = config["n_layers"]
-        n_mult           = config["multiple_of"]
+        #n_mult           = config["multiple_of"]
```
Comment (Cebtenzzre) [inline convert.py:247]: do we need this line at all?

Comment (Green-Sky) [reply convert.py:247]: no, let me remove some more.

**Comment by goerch** [comment]
Hm. A layman's view: I know of `multiple_of` as an input to compute the hidden dimensions, is this mapped to `ffn_dim_multiplier` now? I'd find it easier to follow if you'd be going to support different model classes to back reference the original models.

**Review by cebtenzzre — APPROVED** [review]

File: convert.py [hunk 2]
```diff
@@ -250,15 +236,14 @@ def loadHFTransformerJson(model: LazyModel, config_path: Path) -> Params:
         )
 
     # LLaMA v2 70B params.json
-    # {"dim": 8192, "multiple_of": 4096, "ffn_dim_multiplier": 1.3, "n_heads": 64, "n_kv_heads": 8, "n_layers": 80, "norm_eps": 1e-05, "vocab_size": -1
+    # {"dim": 8192, "ffn_dim_multiplier": 1.3, "n_heads": 64, "n_kv_heads": 8, "n_layers": 80, "norm_eps": 1e-05, "vocab_size": -1
```
Comment (Cebtenzzre) [inline convert.py:239]: Actually, why are we removing this?

Comment (Green-Sky) [reply convert.py:239]: because it became irrelevant, but I guess removing it form the example is a bad idea.

**Comment by Green-Sky** [comment]
> Hm. A layman's view: I know of `multiple_of` as an input to compute the hidden dimensions, is this mapped to `ffn_dim_multiplier` now? I'd find it easier to follow if you'd be going to support different model classes to back reference the original models.

I am not sure where they come from, but yea, they AND the formulas describe a relationship in the architecture. But `multiple_of` can be a different value too, there are always multiple solutions. They also only seem to be specific to llama. falcon for example just uses `4 * hidden_size` . Huggingface models also don't contain the information at all, they just have the `intermediate_size` param.

**Pull request merged**